docs/ref/databases.txt

```diff
@@ -1007,14 +1014,22 @@ using RAC or pluggable databases without ``tnsnames.ora``, for example.
 
 Example of an Easy Connect string::
 
-    'NAME': 'localhost:1521/orclpdb1',
+    {
+        ...: ...,
+        "NAME": "localhost:1521/orclpdb1",
+        ...: ...,
```

Comment:
```suggestion
        # ...
        "NAME": "localhost:1521/orclpdb1",
        # ...
```
As used before maybe can work something like that?

Reply:
This works, thanks @pauloxnet !!